app/Http/Middleware/RequirePassword.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+class RequirePassword extends \Illuminate\Auth\Middleware\RequirePassword
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next, $redirectToRoute = null)
+    {
+        if(!$request->user()->hasPassword()){
```

Comment:
Don't think so, because it will only be used when a user is fully authed (i.e. it should also be behind the auth middleware). Might add in a check to make sure `$request->user()` isn't null though